pkgs/desktops/gnome-3/extensions/text-translator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, gnome3, pkgs, substituteAll, translate-shell}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-text-translator";
+  version = "unstable-2020-02-25";
+
+  src = fetchFromGitHub {
+    owner = "gufoe";
+    repo = "text-translator";
+    rev = "991daadff538bd7acd15cf60415704e0215f2bf6";
+    sha256 = "098a967p1gr8g20nz0fdnlmpc6k40fh0ijqcdhp7ckfqr39imkhr";
+  };
+
+  uuid = "text_translator@awamper.gmail.com";
+
+  patches = [
+    (substituteAll {
+      src = ./fix-trans.patch;
+      trans = "${translate-shell}/bin/trans";
+    })
+  ];
+
+  installPhase = ''
```

Comment:
In your other PR you did add the hooks
runHook preInstall
runHook postInstall

Reply:
Done!